Makefile

```diff
@@ -31,6 +31,7 @@ vet:
 	@scripts/run-go-vet.sh
 
 .PHONY: gotools
+gotools: export GO111MODULE=off
```

Comment:
Not sure about this one ... but it's either this or go with the tools strategy from the other repos as you can't add this as a dep in go.mod (it will be removed when running `go mod tidy`.

Reply:
Since we don't need a VM, I'd suggest moving to a container. Then you can use the tools pattern in a manner that is similar to [this](https://github.com/hyperledger/fabric-chaincode-go/tree/master/ci).

If you want to keep the VM, you'll need the tools pattern and install from the deps.

Reply:
I've seen other projects do it this way as well ... any reason this won't work?  Other than pinning to specific version of goimports?

Reply:
We've been burned a few times by having unversioned deps for tools. The behaviour changes and all of a sudden all of the PR's start breaking, then we have to fix all of the issues and get everyone to rebase. Granted this a low traffic repo, it definitely thepattern to follow.